src/Utils.js

```diff
@@ -78,107 +78,40 @@ export function getRules(featureTypeStyle, feature, resolution, options = {}) {
 }
 
 /**
- * Go through all rules with an external graphic matching the image url
- * and update the __loadingState metadata for the symbolizers with the new imageLoadState.
- * This action replaces symbolizers with new symbolizers if they get a new __loadingState.
- * @param {object} featureTypeStyle A feature type style object.
- * @param {string} imageUrl The image url.
- * @param {string} imageLoadState One of 'IMAGE_LOADING', 'IMAGE_LOADED', 'IMAGE_ERROR'.
+ * Function to memoize style conversion functions that convert sld symbolizers to OpenLayers style instances.
```

Comment:
only used in /styles (maybe it should be there somewhere)

Reply:
I've moved style-related utility functions to a separate styleUtils.js module in bde7add.